tests/kafkatest/tests/core/transactions_upgrade_test.py

```diff
@@ -179,7 +179,7 @@ def copy_messages_transactionally_during_upgrade(self, input_topic, output_topic
 
         self.perform_upgrade(from_kafka_version)
 
-        copier_timeout_sec = 180
+        copier_timeout_sec = 360 
```

Comment:
```suggestion
        copier_timeout_sec = 360
```
sorry again this something for SCA. Taking away the off-topics upfront.

spotless and rewrite both ready to fix on their own.